packages/expo-sqlite/ios/SQLAction.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+enum SQLAction: String, Enumerable {
+  case insert
+  case delete
+  case update
+  case unknown
+
+  static func fromCode(value: Int32) -> SQLAction {
+    switch value {
```

Comment:
What are the other cases? Can we maybe link to a documentation where we can find what these "magic" numbers mean? I think it's especially important that there are no specific constants which names would already mean something.

Reply:
There are 34 of them 😅 In our case, for the update hook, these are the ones we are interested in but the full list is here https://www.sqlite.org/c3ref/c_alter_table.html